### ISSUE TITLE
View width and height not correct. But if you resize it gets fixed. 

### ISSUE THREAD

**Issue by max-koa** [AR-js-org/AR.js#54]
**Do you want to request a feature or report a bug?**
Possible bug

**What is the current behavior?**
Sometimes, not always, when I open in the browser, the camera view will be small. If I open and close the console, then it will cover the whole browser (as it was supposed to). A small fix that works some times, is to set a timer, around 2 seconds, and resize after the 2 seconds.

**If the current behavior is a bug, please provide the steps to reproduce.**
I am using wamp as a server. By opening the browser it happens.

**What is the expected behavior?**
It is expected that the camera view will be the size of the complete view width and height from the beginning.

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Operating System and** Device Name
Chrome Version 80.0.3987.132
Windows 10.
Also tried in Mozilla latest version and it happens the same.

**Comment by kalwalt** [comment]
Yes we know the issue, we want to fix this. This seems not occuring in Mobile device, desktop is affected. As you said you need to add a setTimeout function as in the basic example:

https://github.com/AR-js-org/AR.js/blob/57e7a2864bd59f2b4dec03c6e99e72aeca212ea8/three.js/examples/basic.html#L61-L65

we should try a more robust solution in my opinion. without adding that code inside the `<script>` tags.

**Comment by max-koa** [comment]
Another solution, that in my case it does not affect performance, is to add the resize function inside the update function. It resizes every time it updates, but this way, it surely resizes and fits the whole screen. For some reason, the setTimeOut fails sometimes.

**Comment by JamesBotterill** [comment]
not sure but this might be a similar issue

**Comment by JamesBotterill** [comment]
@kalwalt  

As mentioned in issue #74 

I'm not sure if it's the same issue, gif uploaded to show issue

![ezgif-3-23cab09f419a](https://user-images.githubusercontent.com/9433760/79877106-be323e00-83e3-11ea-9725-c322653770da.gif)

**Comment by nicolocarpignoli** [comment]
might be related to: https://github.com/AR-js-org/AR.js/issues/26

**Comment by max-koa** [comment]
I'm not sure if this was happening before. But currently the trick is not working as expected. Though it resizes and it fills the whole screen, the image is now larger than the actual screen. Now I can scroll up and down, and even though I try to set the height and width to 100%, it will update and set it back to the original size (which is larger than te actual screen).

**Comment by max-koa** [comment]
> might be related to: #26

Not sure if they are related. The reason behind this is that I am using the laptops webcam and I have no other cameras connected to it. Not using smartphone for this.
